test-validator/src/lib.rs

```diff
@@ -526,7 +531,40 @@ impl TestValidatorGenesis {
         mint_address: Pubkey,
         socket_addr_space: SocketAddrSpace,
     ) -> Result<TestValidator, Box<dyn std::error::Error>> {
-        TestValidator::start(mint_address, self, socket_addr_space).map(|test_validator| {
+        TestValidator::start(
```

Comment:
You can remove this code and forward to start_with_mint_address_and_geyser_plugin_rpc with None for rpc_to_plugin_manager_receiver

Reply:
done